k2/csrc/fsa_equivalent.cc

```diff
@@ -169,8 +169,11 @@ bool IsRandEquivalent(const Fsa &a, const Fsa &b, std::size_t npath /*=100*/) {
 
 template <FbWeightType Type>
 bool IsRandEquivalent(const Fsa &a, const float *a_weights, const Fsa &b,
-                      const float *b_weights, bool top_sorted /*=true*/,
-                      std::size_t npath /*= 100*/) {
+                      const float *b_weights, float beam /*=kFloatInfinity*/,
```

Comment:
will implement the version for RmEpsilonPrunedLogSum in another PR(after implement RmEpsilonPrunedLogSum)

Reply:
cool